metaci/build/flows.py

```diff
@@ -37,8 +41,10 @@ def post_task(self, step, result):
             flowtask.status = "error"
         else:
             flowtask.status = "complete"
-
         flowtask.save()
-
         if "robot_outputdir" in result.return_values:
-            import_robot_test_results(flowtask, result.return_values["robot_outputdir"])
+            test_results = import_robot_test_results(
+                flowtask, result.return_values["robot_outputdir"]
+            )
+            if settings.RESULT_EXPORT_ENABLED:
+                export_robot_test_results(flowtask, test_results)
```

Comment:
We need to put this in a try/except block that logs exceptions but doesn't block continued execution of the flow.